hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -427,6 +443,14 @@ public boolean isFlushEnabled() {
     return this.enableFlush;
   }
 
+  public boolean isAppendWithFlushEnabled() {
+    return this.enableAppendWithFlush;
+  }
+
+  public boolean isOutputStreamFlushDisabled() {
```

Comment:
Many changes from trunk merge is showing up as new updates even though not added as part of this PR.